x-pack/plugins/fleet/public/components/cloud_formation_guide.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { Link } from 'react-router-dom';
+
+export const CloudFormationGuide = () => {
+  return (
+    <EuiText>
+      <p>
+        <FormattedMessage
+          id="xpack.fleet.cloudFormation.guide.description"
+          defaultMessage="CloudFormation will create all the necessary resources to evaluate the security posture of your AWS environment. {learnMore}."
+          values={{
+            learnMore: (
+              <Link to={'https://ela.st/cspm-get-started'}>
```

Comment:
same here, we should probably link to cloudformation guide if we have that, maybe even something from amazon? 
also consider to have this link in a shared const since its being used both here and in the pop up modal

Reply:
@opauloh for this modal, let's link to the AWS CloudFormation documentation page you point out above. We can optimize this to link to our documentation page in a more sustainable way at a later point. One change to make is, instead of it being just Learn more, make it says `Learn more about CloudFormation`